tutorials/milvus_lite_arize.ipynb

```diff
@@ -0,0 +1,1428 @@
+{
```

Comment:
<p>I still need to add this pre-commit hook, but can you install <code>nbqa</code> and <code>ruff</code> and run <code>nbqa ruff --fix /path/to/notebook.ipynb</code> before merging the real deal? Among other things, that will order the imports and remove unused imports.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>